### ISSUE TITLE
Update some "Vorstand" images

### ISSUE THREAD

**Issue by bixel** [pep-dortmund/homepage#112]
See the comments in #104

**Comment by hmoldenhauer** [comment]
only two more images needed, than this issue can be closed

**Comment by beckstev** [comment]
We still need one picture from Julian. 
Should I write him an Email and ask for the image?

**Comment by hmoldenhauer** [comment]
Yes. Maybe you have more luck than me. I already tried is several times ;)
